src/netconf.c

```diff
@@ -300,7 +301,11 @@ np2srv_rpc_get_cb(sr_session_ctx_t *session, uint32_t UNUSED(sub_id), const char
 
     /* perform correct NACM filtering */
     sr_session_get_orig_data(session, 1, NULL, (const void **)&username);
-    ncac_check_data_read_filter(&data_get, username);
+    err_info = sr_nacm_check_data_read_filter(&data_get, username);
+    if (err_info) {
+        rc = SR_ERR_INTERNAL;
```

Comment:
No need to set your own err code, just use the one in `err_info`. More similar cases follow.